zproject/settings.py

```diff
@@ -495,6 +495,11 @@ def get_secret(key):
     # Also we auto-generate passwords for the default users which you
     # can query using ./manage.py print_initial_password
     INITIAL_PASSWORD_SALT = get_secret("initial_password_salt")
+else:
+    # Use best password hashing algorithm argon2 for PRODUCTION
```

Comment:
If the algorithm is called "Argon2" then - for ease of grepping - we should call it that in this comment as well:

> For production, use the best password hashing algorithm: Argon2